Assets/Tests/PlayModeTests/Scenarios/UP_UserProfile.cs

```diff
@@ -0,0 +1,410 @@
+                { "StringList", new List<string> { "a", "b", "c" } }
+            };
+
+            Countly.Instance.UserProfile.SetProperties(userProperties);
+        }
+
+        private void SendUserData()
+        {
+            Countly.Instance.UserProfile.SetProperty("a12345", "My Property");
+            Countly.Instance.UserProfile.Increment("b12345");
+            Countly.Instance.UserProfile.IncrementBy("c12345", 10);
+            Countly.Instance.UserProfile.Multiply("d12345", 20);
+            Countly.Instance.UserProfile.Max("e12345", 100);
+            Countly.Instance.UserProfile.Min("f12345", 50);
+            Countly.Instance.UserProfile.SetOnce("g12345", "200");
+            Countly.Instance.UserProfile.PushUnique("h12345", "morning");
+            Countly.Instance.UserProfile.Push("i12345", "morning");
+            Countly.Instance.UserProfile.Pull("k12345", "morning");
+        }
+
+        private Dictionary<string, object> ExpectedUserData()
+        {
+            var expectedUserDetails = new Dictionary<string, object>
+            {
+                { "a12345", "My Property" },
+                { "b12345", new Dictionary<string, object> { { "$inc", 1.0 } } },
+                { "c12345", new Dictionary<string, object> { { "$inc", 10.0 } } },
+                { "d12345", new Dictionary<string, object> { { "$mul", 20.0 } } },
+                { "e12345", new Dictionary<string, object> { { "$max", 100.0 } } },
+                { "f12345", new Dictionary<string, object> { { "$min", 50.0 } } },
+                { "g12345", new Dictionary<string, object> { { "$setOnce", "200" } } },
+                { "h12345", new Dictionary<string, object> { { "$addToSet", new List<string> { "morning" } } } },
+                { "i12345", new Dictionary<string, object> { { "$push", new List<string> { "morning" } } } },
+                { "k12345", new Dictionary<string, object> { { "$pull", new List<string> { "morning" } } } }
+            };
+
+            return expectedUserDetails;
+        }
+
+        private void SendSameData()
+        {
+            Countly.Instance.UserProfile.SetProperty("a12345", "1");
+            Countly.Instance.UserProfile.SetProperty("a12345", "2");
+            Countly.Instance.UserProfile.SetProperty("a12345", "3");
+            Countly.Instance.UserProfile.SetProperty("a12345", "4");
+        }
+
+        private Dictionary<string, object> ExpectedSameData()
+        {
+            // Example expected user details for validation
+            var expectedUserDetails = new Dictionary<string, object>
+            {
+                { "a12345", "4" }
+            };
+
+            return expectedUserDetails;
+        }
+
+        // UserProfile calls in Countly.Instance.UserProfile
+        // We initialize SDK without consent requirement, check the RQ and EQ sizes, then call public methods
+        // Methods we call should record nothing in the RQ and EQ
+        [Test]
+        public void UP_200_CNR_A()

```

Comment:
maybe expected output might be added
![image](https://github.com/user-attachments/assets/b2a36625-4665-4b0e-a471-09e1546c2963)

Reply:
yes cool addition adding now